packages/admin-date-time/src/core/DatePicker.tsx

```diff
@@ -0,0 +1,151 @@
+import "react-dates/initialize";
+import "react-dates/lib/css/_datepicker.css";
+
+import { ClearInputButton, InputBase } from "@comet/admin";
+import { ClickAwayListener, InputBaseProps, Paper, Popper, Typography, WithStyles } from "@material-ui/core";
+import { withStyles } from "@material-ui/styles";
+import * as moment from "moment";
+import * as React from "react";
+import { DayPickerSingleDateController, DayPickerSingleDateControllerShape } from "react-dates";
+import { FieldRenderProps } from "react-final-form";
+import { useIntl } from "react-intl";
+
+import { getDateControllerText } from "../utils/getDateControllerDate";
+import styles from "./DatePicker.styles";
+
+export interface DatePickerThemeProps extends InputBaseProps {
+    numberOfMonths?: number;
+    showClearButton?: boolean;
+    dateControllerProps?: DayPickerSingleDateController;
+}
+
+const Picker: React.FC<WithStyles<typeof styles> & DatePickerThemeProps & FieldRenderProps<Date, HTMLInputElement>> = ({
+    classes,
+    input,
+    disabled,
+    placeholder,
+    fullWidth = false,
+    showClearButton,
+    dateControllerProps,
+    inputProps,
+    ...restProps
+}) => {
+    const intl = useIntl();
+    const localeName = intl.locale;
+    const locale = moment().locale(localeName);
+    const localeDateFormat = locale.localeData().longDateFormat("L");
+    const placeholderText = placeholder
+        ? placeholder
+        : intl.formatMessage({ id: "cometAdmin.dateTime.datePicker.placeholder", defaultMessage: "Date" });
+
+    React.useEffect(() => {
+        moment.locale(localeName);
+    }, [localeName]);
+
+    const { value, onChange, onBlur, onFocus, ...restInput } = input;
+    const rootRef = React.useRef(null);
+    const [showPopper, setShowPopper] = React.useState<boolean>(false);
+    const [showDayPicker, setShowDayPicker] = React.useState<boolean>(false);
+    const [customValue, setCustomValue] = React.useState<string | null>(null);
```

Comment:
das versteh ich nicht ganz: was für problem löst du mit customValue?

Reply:
Im Input wird normalerweise `input.value` ausgegeben. Wenn manuell Text reingeschrieben wird, muss dieser unvalidiert angezeigt werden, wie bei einem normalen Text-Input.

Nachdem das ein nicht-validierter Text ist, kann/soll er nicht mit `onChange` gespeichert werden, desswegen wird er in `customValue` zwischengespeichert. Bei `onBlur` wird dieser Text dann validiert/geparsd und dann mit `onChange` in `input.value` gespeichert.